src/check_str.c

```diff
@@ -127,9 +127,8 @@ static int percent_passed(TestStats * t)
 {
     if(t->n_failed == 0 && t->n_errors == 0)
         return 100;
-    else if(t->n_checked == 0)
```

Comment:
No it is very small modification, so I putted it with cppcheck. `else` is useless as `return` in the previous exit the function if the test is true.